clojure-mode.el

```diff
@@ -1009,7 +1009,7 @@ nil."
                     (one-or-more (any whitespace "\n")))
       ;; why is this here? oh (in-ns 'foo) or (ns+ :user)
       (zero-or-one (any ":'"))
-      (group (one-or-more (not (any "()\"" whitespace))) word-end)))
+      (group (one-or-more (not (any "()\"" whitespace))) symbol-end)))
```

Comment:
Not sure how correct our symbol definition is. If memory serves - not very correct. This change definitely requires unit tests illustrating it actually solves something.